assets/src/components/activities/AuthoringElement.tsx

```diff
@@ -51,6 +51,10 @@ export abstract class AuthoringElement<T extends ActivityModelSchema> extends HT
     };
   }
 
+  transformModel(model: any): T {
```

Comment:
I understand this to be the inline model versioning upgrade, but the name "transformModel" is confusing because activities already have a notion of "transformed models" (for delivery purposes).  Can we change this name to be "migrateModelVersion" ? or something similar?